restaurant_data_clean/data_clean.R

```diff
@@ -0,0 +1,53 @@
+rm(list=ls())
+
+pacman::p_load(sp, rgdal, data.table, dplyr, dtplyr)
+
+load_foodspdf <- function(method="wget"){
+    base <- "ftp://ftp.kingcounty.gov/gis-web/GISData/"
+    dataurl <- paste0(base, "restaurant_inspections_SHP.zip")
```

Comment:
We might wanna archive this, but I  like that the function pulls it from the source